prisma/schema.prisma

```diff
@@ -128,7 +128,7 @@ model Submission {
   isCorrect        Boolean[]      @map(name: "is_correct")
   randomSeed       Float[]        @map(name: "random_seed")
 
-  @@unique([puzzleInstanceId, userId])
+  @@unique([puzzleInstanceId, userId], map: "puzzleInstanceId_userId")
```

Comment:
Do we need this? I thought this would be the default map name

Reply:
it got error when I check this function, so I added this ( I think eslint did that)